helpers/tools.ts

```diff
@@ -46,6 +53,14 @@ export const supportedTools: Tool[] = [
       },
     ],
     supportedFrameworks: ["fastapi", "express", "nextjs"],
+    type: ToolType.LLAMAHUB,
+  },
+  {
+    display: "Weather",
+    name: "weather",
```

Comment:
Do we have toolspec in TS, or just function tool for now? How about we keep adding the `function.` prefix to the name, which would make it easier for the factory?

Reply:
@leehuwuj don't quite understand that - can't you just pass the llamahub part to the llamahub factory?